Tests/Remove-Registry.Tests.ps1

```diff
@@ -0,0 +1,25 @@
+Import-Module -Force $PSScriptRoot/../Docker.Build.psm1
```

Comment:
Not an actual error for the line, but its filename, `Remove-Prefix.Tests.ps1` you mean? 😄